### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by yankev**
Updated `plotly.min.js` 
Updated `plot_schema`
Added `update_offline` in makefile in order to automatically update `plotly.min.js`
Replaced deprecated packages with newer ones in `widgets.py`

### CODE REVIEW

**Comment by cldougl** [comment]
@yankev 
reasoning for not auto updating `plotly.min.js`
https://github.com/plotly/plotly.py/pull/348#discussion_r44943281

**Comment by yankev** [comment]
@Chelsea: finally!!!

**Comment by cldougl** [comment]
🎉 I'll take a look :)

File: makefile [hunk 1]
```diff
@@ -57,3 +57,16 @@ pull_chunked : submodules/chunked_requests
...
+	@echo "------------------"
+	python -c "import urllib2;\
+				cdn_url = 'https://cdn.plot.ly/plotly-latest.min.js';\
+				response = urllib2.urlopen(cdn_url);\
+				html = response.read();\
+				f = open('./plotly/offline/plotly.min.js', 'w');\
+    		 	f.write(html);\
+				f.close()"
+	@echo "---------------------------------"
+	@echo "Remember to update the CHANGELOG!"
```
Comment (cldougl) [inline makefile]: :D yes update that changelog!!

File: makefile [hunk 2]
```diff
@@ -57,3 +57,16 @@ pull_chunked : submodules/chunked_requests
 	@echo ""
 	@echo "Pulling down updates from chunked_requests"
 	cd submodules/chunked_requests; git pull origin master
+
+update_offline :
```
Comment (cldougl) [inline makefile]: can we name this to be a tiny bit more informative `update_plotlyjs_for_offline` ?

**Comment by cldougl** [comment]
@yankev ahh can you bump the version number!
Then you can 💃

**Pull request merged**